bin/spark.js

```diff
@@ -12,6 +12,7 @@ import assert from 'node:assert'
 const {
   PORT = 8080,
   HOST = '127.0.0.1',
+  DOMAIN = 'api.filspark.com',
```

Comment:
On the second thought, maybe we should configure this via Fly config file and don't assume any default domain for people running this via `npm start`?

Reply:
Alternatively, define domain as `localhost` by default, so that we can assume domain is always configured.

Then add an environment variable to our Fly config to change this to the domain used by our deployment.

Reply:
Good point, I hadn't thought about dev mode

Reply:
Let's default to `localhost` and set an env var